src/log/Log.cc

```diff
@@ -354,9 +354,8 @@ void Log::_flush(EntryQueue *t, EntryQueue *requeue, bool crash)
 void Log::_log_message(const char *s, bool crash)
 {
   if (m_fd >= 0) {
-    int r = safe_write(m_fd, s, strlen(s));
-    if (r >= 0)
-      r = safe_write(m_fd, "\n", 1);
+    std::string b {s, '\n'};
```

Comment:
I'm trying to match this initializer to the std::string reference at http://en.cppreference.com/w/cpp/string/basic_string/basic_string, and the only one I'm seeing I think is interpreting '\n' as an integer for "count", rather than as something to be appended. (Constructor 4)
Am I missing something here?